.gitignore

```diff
@@ -1,4 +1,12 @@
 node_modules
 *.log
 dist
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
do we need these? pnp/yarn shouldn't exist in this repo so shouldn't cause any of these files to exist?